compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1918,8 +1918,23 @@ static LogicalResult setConvRootConfig(mlir::FunctionOpInterface entryPointFn,
   int64_t numTilingDims = parallelTileSizes.size();
   tileSizes.emplace_back(numTilingDims, 0);
 
+  // Set "scalable" flags
+  ScalableTileFlagsListType scalableTileFlags;
+  auto targetAttr = IREE::HAL::ExecutableTargetAttr::lookup(entryPointFn);
+  if (isAArch64(targetAttr) && clEnableScalableVectorization) {
+    // Level 1: Distribution
+    scalableTileFlags.emplace_back(numTilingDims, false);
+    // Level 2: Parallel
+    scalableTileFlags.emplace_back(
+        SmallVector<bool>{false, false, false, true, false, false});
```

Comment:
Are we able to use `linalg::inferConvolutionDims` and set `true` on the specific field? It will help a lot if we always want to make the dimension (e.g., N/H/W dim) scalable. If it is not always the same dim (for different conv ops), please update the if-check with allowlist. E.g.,

```
if (isAArch64(targetAttr) && clEnableScalableVectorization
  && isa<linalg::DepthwiseConv1DNwcWcOp>(op)) {
 // ...
}
```

Reply:
> Are we able to use linalg::inferConvolutionDims 

Not yet:
* https://github.com/llvm/llvm-project/pull/90057

Otherwise, yes, thanks for the suggestions! :)